deletequestion.php

```diff
@@ -91,4 +91,4 @@
 <form action="" method="post">
   <input class="button-input" type="submit" name="delete" value="Delete">
   <input class="button-input" type="submit" name="go_back" value="Go Back">
-</form>
\ No newline at end of file
+</form>
```

Comment:
Nepochopil jsem tento coment - `// Kontrola, zda je uživatel povolený`

Ten while je zde zbytečný, mělo by stačit uložit si výsledek query do `$row` a pak to jen ifnout jak to děláš :+1: 
```
while ($row = $result->fetch_assoc()) {
      if ($row['dog_id'] == $dog_id) {
        $dogFound = true;
        break;
      }
    }
```

Uvnitř toho ifu na řádku 46 podle mě nemůže dojít k else na ř. 61. Ten check jsi provedla jižtouto query: `$sql = "SELECT dog_id FROM dogs WHERE dog_owner_id = ?";`